### ISSUE TITLE
the loss is different from rasa?

### ISSUE THREAD

**Issue by datalee** [cheesama/DIET-pytorch#10]
MarginRankingLoss or CrossEntropyLoss？

**Comment by cheesama** [comment]
I used CrossEntropyLoss
